javacutil/src/main/java/org/checkerframework/javacutil/PluginUtil.java

```diff
@@ -34,8 +34,10 @@
     public static final String JAVAC_PATH_OPT = "-javacJar";
 
     /**
-     * Option name for specifying an alternative jdk.jar location. The accompanying value MUST be
-     * the path to the jar file (NOT the path to its encompassing directory)
+     * Option name for specifying an alternative jdk.jar location in case of Java8. The accompanying
+     * value MUST be the path to the jar file (NOT the path to its encompassing directory) In Case
```

Comment:
Put periods at ends of sentences.  Put "if" at beginning of sentences -- you don't want the reader to read an entire sentence only to learn at the end that it is irrelevant.  I would put the two sentences in different paragraphs to make the structure easier to read.

Reply:
Done